.github/workflows/slow.yml

```diff
@@ -42,3 +43,10 @@ jobs:
       - name: Test with pytest
         run: |
           py.test -m slow
+          slack_api:
```

Comment:
What is this line doing?  I think we can safely remove it - I don't see anything like this in the up9cloud/action-notify docs.